extensions/github/package.json

```diff
@@ -59,6 +60,12 @@
       "description": "Create an issue in one of your GitHub repositories.",
       "mode": "view"
     },
+    {
+      "name": "create-issue-branch",
+      "title": "Create Issue Branch",
```

Comment:
is "Issue branch" something that people know? I've never seen it before. How about having just `Create Branch` and have the `Issue` dropdown be optional?

Reply:
If I'm understanding you correctly, you mean to make issueId optional in branch form? Currently, when creating a branch, an issueId is required.

Reply:
Is it? You can create a new ref: https://docs.github.com/en/rest/git/refs?apiVersion=2022-11-28#create-a-reference without an issueID. It's only to create an "issue branch" that you need one.

Reply:
Oh I haven't checked their REST API.